lib/ReactViews/FeatureInfo/FeatureInfoSection.jsx

```diff
@@ -440,16 +446,52 @@ function mustacheFormatNumberFunction() {
         const startsWithdoubleBraces = (text.length > 4) && (text[0] === '{') && (text[1] === '{');
         if (!components || startsWithdoubleBraces) {
             // If no options were provided, just use the defaults.
-            return formatNumberForLocale(render(text));
+            return customProcessing(render(text));
         }
         // Allow {foo: 1} by converting it to {"foo": 1} for JSON.parse.
         const quoteReg = /([{,])(\s*)([A-Za-z0-9_\-]+?)\s*:/g;
         const jsonOptions = components[1].replace(quoteReg, '$1"$3":');
         const options = JSON.parse(jsonOptions);
-        return formatNumberForLocale(render(components[2]), options);
+        return customProcessing(render(components[2]), options);
     };
 }
 
+/**
+ * Returns a function which implements number formatting in Mustache templates, using this syntax:
+ * {{#terria.formatNumber}}{useGrouping: true}{{value}}{{/terria.formatNumber}}
+ * @private
+ */
+function mustacheFormatNumberFunction() {
+    return mustacheJsonSubOptions((text, options) => {
+        return formatNumberForLocale(text, options);
+    });
+}
+
+/**
+ * Returns a function which implements date/time formatting in Mustache templates, using this syntax:
+ * {{#terria.formatDateTime}}{dateFormatString: "npm dateFormat string"}Date_Expression{{/terria.formatDateTime}}
+ * dateFormatString If present, will override the default date format (see https://www.npmjs.com/package/dateformat)
+ * Eg. "isoDateTime" or "dd-mm-yyyy HH:MM:ss".
+ * If the Date_Expression can't be parsed using Date.parse() it will be used(returned) unmodified by the terria.formatDateTime section expression.
+ * If no valid date formatting options are present in the terria.formatDateTime section isoDateTime will be used.
+ * @private
+ */
+function mustacheFormatDateTime() {
+    return mustacheJsonSubOptions((text, options) => {
+        var date = Date.parse(text);
+
+        if (!defined(date) || isNaN(date)){
+            return text;
+        }
+
+        if (defined(options) && defined(options.dateFormatString)) {
```

Comment:
I think simply `format` might be better than `dateFormatString`. The `date` bit is implied by the fact that you're passing this it a thing called `formatDateTime` (and perhaps it's even a bit misleading because it's also a time format, not just a date format). And it's not really necessary to state the type (`string`).